src/languages/stan.js

```diff
@@ -523,18 +386,26 @@ export default function(hljs) {
         keywords: 'upper'
       },
       {
-        className: 'keyword',
-        begin: /\btarget\s*\+=/
+        // hack: in range constraints, upper must follow either , or <
+        // <multiplier = ..., offest = ...> or <offset = ...>
+        begin: /[<,]\s*offset\s*=/,
+        keywords: 'offset'
       },
       {
-        begin: '~\\s*(' + hljs.IDENT_RE + ')\\s*\\(',
-        keywords: DISTRIBUTIONS
+        // hack: in range constraints, upper must follow either , or <
+        // <offset = ..., multiplier = ...> or <multiplier = ...>
+        begin: /[<,]\s*multiplier\s*=/,
+        keywords: 'multiplier'
+      },
+      {
+        className: 'keyword',
+        begin: /\btarget\s*/,
```

Comment:
If the `+=` specificity isn't needed can this not just be moved into the normal keyword list?

Reply:
`+=` has to be there